src/gdcdictionary/schemas/other_clinical_attribute.yaml

```diff
@@ -550,6 +550,18 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  fertility_history:
+    $ref:
+      - "_terms.yaml#/fertility_history/common"
+    enum:
+      - Conceived > or = 1 child by assisted reproduction
+      - Conceived > or = 1 child by natural conception
+      - Conceived > or = 1 child by unspecified method
```

Comment:
probably above my pay grade, but we use ">=" in other enums (though slightly different contexts) -- should we try to be consistent? and are we open to other phrasings, e.g. "1 or more", "1+", "at least 1"

Reply:
Good point!  My thought would be that the following would be acceptable: 

- Conceived 1+ child by assisted reproduction
- Conceived at least 1 child by assisted reproduction

@tlicht3 do you have any thoughts?  Are there guidelines that should be followed?